docs/operating-scylla/nodetool-commands/status.rst

```diff
@@ -18,84 +18,93 @@ Example output:
+|          |                                                               |
++----------+---------------------------------------------------------------+
+|Status    |``U`` - The node is up.                                        |
+|          |                                                               |
+|          |``D`` - The node is down.                                      |
+|          |                                                               |
+|          |``X`` - The node is :ref:`excluded <status-excluded>`.         |
++----------+---------------------------------------------------------------+
+|State     |``N`` - Normal                                                 |
+|          |                                                               |
+|          |``L`` - Leaving                                                |
+|          |                                                               |
+|          |``J`` - Joining                                                |
+|          |                                                               |
+|          |``M`` - Moving                                                 |
++----------+---------------------------------------------------------------+
+|Address   |The IP address of the node.                                    |
+|          |                                                               |
++----------+---------------------------------------------------------------+
+|Load      |The size on disk the ScyllaDB data                             |
+|          | takes up (updates every 60 seconds).                          |
+|          |                                                               |
+|          |                                                               |
+|          |                                                               |
+|          |                                                               |
++----------+---------------------------------------------------------------+
+|Tokens    |The number of tokens per node.                                 |
+|          |                                                               |
+|          |                                                               |
+|          |                                                               |
++----------+---------------------------------------------------------------+
+|Owns      |The percentage of data owned by                                |
+|          |the node (per datacenter) multiplied by                        |
+|          |the replication factor you are using.                          |
+|          |                                                               |
+|          |For example, if the node owns 25% of                           |
+|          |the data and the replication factor                            |
+|          |is 4, the value will equal 100%.                               |
```

Comment:
Nit: can improve line breaks with the wider table (not critical in this case)

Reply:
Not sure why we need the space. It was like that already. And it is not rendered like that.